atsign-foundation/at_tools
# added sharedKeyEncrypted and pubKeyChecksum to metadata

**- What I did**
- Added encryption shared key and publickey checksum of sharedWith atsign in metadata

**- How I did it**
- added sharedKeyEnc and pubKeyCS to Metadata object
- modified UpdateVerbBuilder to support new metadata
- modified update verb and update meta verb syntax

**- How to verify it**
- run update verb builder test in at_commons
- once at_commons is published, existing version of at_client should work fine without any backward compatibility issue. 

**- Description for the changelog**
- Added encryption shared key and public key checksum to metadata
